pom.xml

```diff
@@ -80,9 +81,20 @@
         </pluginManagement>
 		<plugins>
 			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-			</plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>${spring-boot-maven-plugin.version}</version>
+                <configuration>
+                    <mainClass>${start-class}</mainClass>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>repackage</goal>
```

Comment:
I was coming to the same conclusion myself! Do we actually need to specify the start class though? I note they don't do that here: https://github.com/companieshouse/officer-delta-processor/blob/main/pom.xml.

Reply:
They do here https://github.com/companieshouse/search.api.ch.gov.uk/blob/master/pom.xml and this actually works in tilt and deploys properly

Reply:
Yes, sorry I see you do have a `start-class` property anyway.

Reply:
I should add `officer-delta-processor` does work in tilt and `cidev`.